themes/startwords/assets/scss/_fonts.scss

```diff
@@ -1,127 +1,182 @@
 /* font familes */
 
+/* source-sans-pro-200 - latin */
 @font-face {
     font-family: 'Source Sans Pro';
     font-style: normal;
     font-weight: 200;
-    src: url('/fonts/Source_Sans_Pro/SourceSansPro-ExtraLight.ttf');
+	font-display: swap;
+    src: url('/fonts/Source_Sans_Pro/source-sans-pro-v13-latin-200.woff2') format('woff2'),
```

Comment:
I didn't know we were using local copies of the font, is there a reason for that? Would it be more efficient to load them from google fonts as a CDN?

Reply:
The main reason is that CSS is render-blocking, and google fonts are normally implemented by requesting a stylesheet which then itself connects to a different google domain to get the font ([analysis in this article](https://www.tunetheweb.com/blog/should-you-self-host-google-fonts/)). In some self-hosting situations (maybe our production VMs, haven't tested) it _could_ be a closer call since google's CDN is fast, but here we're essentially using GitHub as a CDN, so that's not an issue.

Reply:
Got it. Thanks for explaining and providing a link.